### PR TITLE
Translation

### PR DESCRIPTION
**Pull request by steins-code**
new translation updates

### CODE REVIEW

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.stringsdict [hunk 1]
```diff
@@ -19,7 +19,7 @@
 			<string>Yesterday, %%1$@</string>
 			<key>other</key>
 			<!-- XFLD: Plural form for Other -->
-			<string>%%2$@, %%1$@</string>
+			<string>%%2$@, %%1$@ Uhr</string>
```
Comment (EisvogelSAP) [inline src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.stringsdict:22]: Nope :-D

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.stringsdict [hunk 2]
```diff
@@ -55,13 +55,13 @@
 			<string>u</string>
 			<key>zero</key>
 			<!-- XFLD: Plural form for Zero -->
-			<string>Last encounter: today</string>
+			<string>%u days since last exposure</string>
 			<key>one</key>
 			<!-- XFLD: Plural form for One -->
-			<string>Last encounter: yesterday</string>
+			<string>%u day since last exposure</string>
```
Comment (EisvogelSAP) [inline src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.stringsdict:61]: I guess this should also have non-breakable spaces

Comment (EisvogelSAP) [reply src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.stringsdict:61]: Same goes for the German equivalent.

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings [hunk 3]
```diff
@@ -80,21 +82,17 @@
...
 
-"ExposureDetection_Explanation_Text_High" = "You have an increased risk of infection because you were last exposed two days ago over a longer period of time and at close proximity to at least one person diagnosed with COVID-19.\n\nTherefore, your risk of infection has been ranked as increased.\n\nWhen you get home, please also avoid close contact with members of your family or household.";
-
-"ExposureDetection_MoreInformation" = "More information";
-
-"ExposureDetection_MoreInformation_URL" = "https://www.google.de";
+"ExposureDetection_Explanation_Text_High" = "You have an increased risk of infection because you were last exposed two days ago over a longer period of time and at close proximity to at least one person diagnosed with COVID-19.\n\nTherefore, your risk of infection has been ranked as increased.\nYour risk of infection is calculated from the risk identification data (duration and proximity) locally on your device. Your risk of infection cannot be seen by, or passed on to, anyone else.\nWhen you get home, please also avoid close contact with members of your family or household.";
 
 "ExposureDetection_Hotline_Number" = "+498007540002";
```
Comment (EisvogelSAP) [inline src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings:97]: Where is the hotline_text key?

Comment (EisvogelSAP) [reply src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings:97]: never mind, I think the hotline is gone from the exposure detection screen. That key can be deleted I guess. Same goes for the German one.

**Review by eisvogel — CHANGES_REQUESTED** [review]

**Review by melloskitten — APPROVED** [review]

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings (hunk 3 above)
Comment (steins-code) [reply src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings]: removed

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.stringsdict (hunk 1 above)
Comment (steins-code) [reply src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.stringsdict]: changed

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.stringsdict (hunk 2 above)
Comment (steins-code) [reply src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.stringsdict]: done

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.stringsdict [hunk 4]
```diff
@@ -149,17 +149,14 @@
 			<string>u</string>
 			<key>zero</key>
 			<!-- XFLD: Plural form for Zero -->
-			<string>Risiko-Ermittlung ist seit weniger als 1 Tag aktiv.
-Eine verlässliche Risiko-Überprüfung kann nur mit einer dauerhaft aktivierten Risiko-Ermittlung erfolgen.</string>
+			<string>Risiko-Ermittlung ist seit weniger als 1&#x00a0;Tag aktiv. Eine verlässliche Risiko-Überprüfung kann nur mit einer dauerhaft aktivierten Risiko-Ermittlung erfolgen.</string>
```
Comment (EisvogelSAP) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.stringsdict:152]: Should this be aligned with the English equivalent? Like "weniger als ein Tag"?

**Review by eisvogel — APPROVED** [review]

**Pull request merged**